dohq_artifactory/admin.py

```diff
@@ -155,7 +155,7 @@ def delete(self):
 class User(AdminObject):
     _uri = "security/users"
 
-    def __init__(self, artifactory, name, email=None, password=None):
+    def __init__(self, artifactory, name, email=None, password=None, disable_ui=True):
```

Comment:
Is this property "true" by default in Artifactory? I guess it should be `False`

Reply:
There is no default in the documentation. I think it is "true" by default.